packages/contentful--react-apps-toolkit/README.md

```diff
@@ -18,7 +18,26 @@ yarn add @contentful/react-apps-toolkit
 
 ## Available features
 
-The following hooks and utilities are exported from the package: Nothing
+The following hooks and utilities are exported from the package:
+
+### `useSDK`
+
+This hook returns the appSDK. You can inject it anywhere in your Contentful app.
+
+The only requirement for using it is that the component that uses it is wrapped within the `SDKProvider`, which ideally should be
```

Comment:
I'd say remove the sentence

"which ideally should be located in the core of your application (e.g. `index.tsx`). "

As long as they only use the hooks inside the provider, it's up to the developer where the provider is located.